docs/java-rest/high-level/ml/start-data-frame-analytics.asciidoc

```diff
@@ -0,0 +1,28 @@
+--
+:api: start-data-frame-analytics
+:request: StartDataFrameAnalyticsRequest
+:response: AcknowledgedResponse
+--
+[id="{upid}-{api}"]
+=== Start Data Frame Analytics API
+
+The Start Data Frame Analytics API is used to start an existing {dataframe-analytics}.
+It accepts a +{request}+ object and responds with a +{response}+ object.
+
+[id="{upid}-{api}-request"]
+==== Start Data Frame Analytics Request
+
+A +{request}+ object requires a {dataframe-analytics} id.
+
+["source","java",subs="attributes,callouts,macros"]
+---------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-request]
+---------------------------------------------------
+<1> Constructing a new start request referencing an existing {dataframe-analytics}
+
+include::../execution.asciidoc[]
+
+[id="{upid}-{api}-response"]
+==== Response
+
+The returned +{response}+ object acknowledges the {dataframe-job} has started.
```

Comment:
`{dataframe-job}` again rather than `{dataframe-analytics}`. Is this intentional have I misunderstood the naming?

Reply:
It is intentional.
{dataframe-job} resolves to "data frame analytics job" and {dataframe-analytics} resolves to "data frame analytics config". So I use the former when I write about the computational process and the latter when I talk about DataFrameAnalyticsConfig.

I may rename {dataframe-analytics} to {dataframe-analytics-config} if you find it clearer. WDYT?

Reply:
Thanks for the explanation. I see you renamed ` {dataframe-analytics}`  to ` {dataframe-analytics-config}` that makes sense to me